.idea/runConfigurations/Run_Migrations.xml

```diff
@@ -0,0 +1,12 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Run Migrations" type="js.build_tools.npm" factoryName="npm">
+    <package-json value="$PROJECT_DIR$/package.json" />
+    <command value="run" />
+    <scripts>
+      <script value="migrate" />
```

Comment:
How exactly does this ```<script value="migrate" />``` work?

Reply:
Nvm I though migrate was a file. I see it now!

Reply:
:+1: